### PR TITLE
Update the price prediction code in terms of validation loss and accuracy matric

### PR DESCRIPTION
**Pull request by kuchiakol**
I have made three changes to the price prediction model.

1. I have added validation loss in LSTM model
  
model = build_lstm_model(
    X_train, output_size=1, neurons=lstm_neurons, dropout=dropout, loss=loss,
    optimizer=optimizer)
history = model.fit(
    X_train, y_train, validation_data=(X_test, y_test), epochs=epochs, batch_size=batch_size, verbose=1, shuffle=True)

2. I have shown the validation training loss and validation loss

import matplotlib.pyplot as plt
plt.plot(history.history['loss'],'r',linewidth=2, label='Train loss')
plt.plot(history.history['val_loss'], 'g',linewidth=2, label='Validation loss')
plt.title('LSTM')
plt.xlabel('Epochs')
plt.ylabel('MSE')
plt.show()

3. Also I have added the accuracy matric in terms of means square error and r2 value

from sklearn.metrics import mean_squared_error
MAE=mean_squared_error(preds, y_test)
MAE

from sklearn.metrics import r2_score
R2=r2_score(y_test, preds)
R2

### CODE REVIEW

**Comment by abhinavsagar** [comment]
LGTM, Thanks for this PR.
